openfisca_uk/variables/finance/benefit/family/universal_credit.py

```diff
@@ -31,11 +31,11 @@ class claims_UC(Variable):
     definition_period = YEAR
 
     def formula(benunit, period, parameters):
-        WTC = benunit("would_claim_WTC", period) & benunit(
-            "is_WTC_eligible", period
+        WTC = benunit("would_claim_wtc", period) & benunit(
```

Comment:
Or `all_` following https://github.com/PolicyEngine/openfisca-tools/issues/28
```suggestion
        WTC = and_(benunit, period, ["would_claim_wtc", "is_wtc_eligible"])
```
Lots of other opportunities to cut code with this new pattern in this PR.